react/features/base/participants/reducer.js

```diff
@@ -511,6 +579,11 @@ function _updateParticipantProperty(state, id, property, value) {
         // not in a conference.
         state.local = set(local, property, value);
 
+        return true;
+
+    } else if (localScreenShare?.id === id) {
```

Comment:
Why not handle remote screen share participants?

Reply:
remote SS participants are also in the `remote` map handled above